packages/emotion-primitives/src/createEmotion.js

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react'
+import reactPrimitives from 'react-primitives'
+import PropTypes from 'prop-types'
+
+import { getStyles } from './getStyles'
+import { convertToRNStyles } from './convertToRNStyles'
+
+const isValidPrimitive = primitive =>
+  ['Text', 'View', 'Image'].indexOf(primitive) > -1
```

Comment:
maybe hoist this constant array above the function to avoid unnecessary garbage collection on this, although probably modern engines should recognize it's constant and hoist it on their own, but it's hard to tell - so I would argue it's better to do that manually